app/pdf/utils.py

```diff
@@ -64,6 +72,7 @@ def make_pdf(atbd: Atbds, major: str, minor: str, filepath: str, auth_data: dict
             page = context.new_page()
             page.goto(atbd_link)
             # wait for a specific marker element to be rendered before generating the PDF
+            time.sleep(5)
```

Comment:
@frozenhelium Weirdly I kept seeing the loading symbol in PDFs while exporting locally. So I added a sleep statement to be on the safe side. Is it possible that we are adding the `#pdf-preview-ready` element prematurely while the loading icon is still showing?

Without `time.sleep`, I see this on each page of the pdf for reference:
![image](https://user-images.githubusercontent.com/1142203/232996840-8f02b934-c425-415e-8d7e-82346f889bfc.png)